src/app/application.cpp

```diff
@@ -740,9 +743,6 @@ void Application::cleanup()
 
 void Application::validateCommandLineParameters()
```

Comment:
Not your code, please help us fix the coding style by adding `const` to function:
`void Application::validateCommandLineParameters() const`

Reply:
Done.